cmd/mint/run.go

```diff
@@ -87,17 +87,11 @@ var (
 
 		},
 		Short: "Start a new run on Mint",
-		Use:   "run [flags] --user-access-token=<token> [task]",
+		Use:   "run [flags] --access-token=<token> [task]",
```

Comment:
I feel like --access-token shouldn't be in this example. Everyone should be running this after running `mint login` and the helptext should take care of explaining how to set an access token manually if you're in CI

Reply:
`--access-token` is also still acceptable (and in some use cases, like in CI, potentially preferable). I think we probably just don't want to leave it here as an expected/seemingly required flag

Reply:
Right, it should be a flag listed in the helptext, it just shouldn’t be in the canonical example. Totally fine to adjust this as a follow up. We should also add an error telling you to run mint login if no access token is set.

Reply:
> We should also add an error telling you to run mint login if no access token is set.

Let's chat about this a little offline